src/components/CommentForm.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+
+import { postComment } from '../__utils__/api';
+import { TextField, Snackbar, Alert } from '@mui/material';
+import LoadingButton from '@mui/lab/LoadingButton';
+
+import SendIcon from '@mui/icons-material/Send';
+
+const CommentForm = ({ user, article_id }) => {
+  const [comment, setComment] = useState('');
+  const [error, setError] = useState(false);
+  const [posting, setPosting] = useState(false);
+  const [open, setOpen] = useState(false);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (comment.length < 1) return setError(true);
+    setPosting(true);
+    postComment(article_id, comment, user).then(() => {
+      setPosting(false);
+      setComment('');
+      setOpen(true);
+    });
+  };
```

Comment:
You should render the post after it has been posted

Reply:
Done but not sure is it the right way
https://github.com/MZHoffman/nc-news/commit/d9495cef245f46f22fc4015840d0dcb5a62f713c
created a state and a function to change it 
when changed trigger useEffect.
invoked from commentForm child component